src/components/SearchBar.tsx

```diff
@@ -242,16 +246,20 @@ export function SearchBar({
   }, [executeEntityPreviewsQuery, renderEntityPreviews, restrictVerticals, universalLimit]);
 
   const handleInputFocus = useCallback((value = '') => {
+    setReadAutocomplete(false);
     answersActions.setQuery(value);
     updateEntityPreviews(value);
     autocompletePromiseRef.current = executeAutocomplete();
   }, [answersActions, autocompletePromiseRef, executeAutocomplete, updateEntityPreviews]);
 
   const handleInputChange = useCallback((value = '') => {
+    if (!readAutocomplete) {
+      setReadAutocomplete(true);
```

Comment:
Is there some way we can put this logic inside of the ScreenReader/Dropdown components? Ideally we don't have to add this logic to current and future components that use Dropdown. The SearchBar already has a LOT of stuff in it